### DISCUSSION TITLE
Connecting to RTL_TCP Server

### DISCUSSION THREAD

**Discussion by YabaiKai** [allangood/rtlamr2mqtt#118] [Q&A] [answered]
This tool is phenomenal! Easy to use, configure, and enjoy.

I'm trying to connect to the RTL_TCP server however and I'm having some trouble. I tried forwarding out the port of the container (default is 1234), but I'm unable to connect. I suppose I might have to set some custom parameters for rtltcp in the config, but the logs tell me the server is started, so unsure.

**Comment by allangood** [comment, marked as answer]
By default, the rtl_tcp server listens on `127.0.0.1`, this way it is not possible to forward requests to it.
If you want it to forward requests to it, you must change the rtl_tcp parameters, like this:
```
custom_parameters:
  rtltcp: "-a 0.0.0.0"
```
And then forward requests to it on your docker command.

Regards.
